apps/rush-lib/src/rush.schema.json

```diff
@@ -149,5 +164,53 @@
     }
   },
   "additionalProperties": false,
-  "required": [ "npmVersion", "rushMinimumVersion", "projects" ]
+  "required": [ "npmVersion", "rushMinimumVersion", "projects" ],
```

Comment:
Did we decide to put this in `rush.json`? I think it may make more sense in a different configuration file under `common/config/rush`.